plugins/woocommerce/client/blocks/assets/js/base/stores/woocommerce/cart.ts

```
@@ -22,680 +24,788 @@ import type {
+                addQueue.clear();
+
+                const requestMap = new Map<number, { type: 'remove' | 'update' | 'add'; key: string; item?: OptimisticCartItem }>();
+                const batchRequests: any[] = [];
+
+                let remainingSlots = MAX_BATCH_SIZE;
+
+                // Combined slicing: take items until total MAX_BATCH_SIZE
+                const processDeletes = currentDeletes.slice(0, remainingSlots);
+                remainingSlots -= processDeletes.length;
+                const remainderDeletes = currentDeletes.slice(processDeletes.length);
+                remainderDeletes.forEach(item => deleteQueue.add(item));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)